lib/tasks/todo.rake

```diff
@@ -0,0 +1,9 @@
+namespace :todo do
+  desc "TODO"
+  desc "Delete items older than seven days"
+
+  task delete_items: :environment do
+    Item.where("created_at <= ?", Time.now - 7.days).destroy_all
+  end
```

Comment:
If you'd wanted to provide test coverage for this, specs can be written for rake tasks (with a sprinkling of rspec configuration):

- config: https://github.com/eliotsykes/rspec-rails-examples/blob/master/spec/support/tasks.rb
- spec: https://github.com/eliotsykes/rspec-rails-examples/blob/master/spec/tasks/subscription_tasks_spec.rb